videos/WoT-Explainer-Video/WOTMotion.fr.vtt

```diff
@@ -0,0 +1,251 @@
+WEBVTT
+
+1
+00:00:00.500 --> 00:00:03.000
+Un producteur fruitier cherche des moyens
+pour augmenter sa productivité 
+
+2
+00:00:03.500 --> 00:00:06.000
+tout en conservant une excellente
+qualité de produits. 
+
+3
+00:00:06.500 --> 00:00:08.000
+Pour surveiller et contrôler
+les conditions de croissance,
+
+4
+00:00:08.500 --> 00:00:13.000
+l'agriculteur prévoit d'installer des capteurs d'humidité et de température,
+et un système d'arrosage.
+
+5
+00:00:13.500 --> 00:00:16.000
+Les données de prévisions météorologiques
+à partir d'un service Web en ligne 
+
+6
+00:00:16.500 --> 00:00:20.000
+seraient également utile pour
+pour prévoir les besoins en arrosage.
+
+7
+00:00:20.500 --> 00:00:25.000
+L'agriculteur décide d'engager
+un consultant informatique. 
+
+8
+00:00:25.500 --> 00:00:29.000
+Il voudrait demander au développeur de son site Web
+de réaliser une solution IdO.
```

Comment:
IoT ? La suite parle de WoT et pas de "WdO" , et l'usage d'"IoT" est courant en français.

Reply:
all translations follow the base line version [here](https://github.com/w3c/wot-marketing/blob/main/videos/WoT-Explainer-Video/WOTMotion.en.vtt)

Reply:
The video does indeed refer to "IoT" and since @vcharpenay confirms that "IoT" is commonly used in French, the french vtt file should use this (and subsequently line 236).